prelude/README.md

```diff
@@ -111,6 +122,16 @@ instance categoryArr :: Category Prim.Function
 ($) :: forall a b. (a -> b) -> a -> b
 ```
 
+Applies a function to its argument
+
+```purescript
+length $ groupBy productCategory $ filter isInStock products =
+length (groupby productcategory (filter isinstock (products)))
```

Comment:
Also, sorry to be pedantic, but this isn't valid PureScript. Can we please separate this into two code sections, and say something like "which is equivalent to:" in between them?